Formula/gnupg.rb

```diff
@@ -72,11 +102,16 @@ def caveats; <<-EOS.undent
       Name-Real: Testing
       Name-Email: testing@foo.bar
       Expire-Date: 1d
+      %no-protection
       %commit
     EOS
-    system bin/"gpg1", "--batch", "--gen-key", "batchgpg"
-    (testpath/"test.txt").write "Hello World!"
-    system bin/"gpg1", "--armor", "--sign", "test.txt"
-    system bin/"gpg1", "--verify", "test.txt.asc"
+    begin
+      system bin/"gpg2", "--batch", "--gen-key", "batchgpg"
```

Comment:
Ideally this would use `gpg`, even though that's just a symlink. Trying to get people to expect that a non-keg-only `gpg` from Homebrew will always be `gpg2`.